rs/tests/driver/src/driver/bootstrap.rs

```diff
@@ -537,56 +528,48 @@ pub fn setup_nested_vms(
     group_name: &str,
     nns_url: &Url,
     nns_public_key: &str,
-    to: &NestedVersionTarget,
 ) -> anyhow::Result<()> {
-    let mut join_handles: Vec<JoinHandle<anyhow::Result<()>>> = vec![];
-    for node in nodes {
-        let t_farm = farm.to_owned();
-        let t_env = env.to_owned();
-        let t_group_name = group_name.to_owned();
-        let t_vm_name = node.name.to_owned();
-        let t_nns_url = nns_url.to_owned();
-        let t_nns_public_key = nns_public_key.to_owned();
-        let t_to = to.to_owned();
-        join_handles.push(thread::spawn(move || {
-            let configured_image =
-                configure_setupos_image(&t_env, &t_vm_name, &t_nns_url, &t_nns_public_key, &t_to)?;
-
-            let configured_image_spec = AttachImageSpec::new(t_farm.upload_file(
-                &t_group_name,
-                configured_image,
-                NESTED_CONFIGURED_IMAGE_PATH,
-            )?);
-            t_farm.attach_disk_images(
-                &t_group_name,
-                &t_vm_name,
-                "usb-storage",
-                vec![configured_image_spec],
-            )?;
+    let mut result = Ok(());
```

Comment:
nit: consider moving it to the bottom where it's used

Reply:
I need to declare somewhere before the `scope` to make it out after collecting the errors.